### PR TITLE
Autopilot Section

### PR DESCRIPTION
**Pull request by Anindyadeep**
This PR adds the section for AutoPilot.

### CODE REVIEW

File: get-started/autopilot.mdx [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+
+Prem AutoPilot is one of our flagship feature which provides the following feature:
+
+1. Automatically chooses the best model which suits the task. 
+2. Automatically enhances the system prompt once a baseline system prompt is provided in the [launchpad](/get-started/launchpad.mdx).
+3. Automatically enhances the prompt template. However this feature needs you to have prompt templates defined in the [Prem Templates section](/get-started/prem-templates). Prompt templates will be enhanced only when they are called through the SDK. From your side you just need to know how to define [templates](/get-started/prem-templates) and use it in [prem-sdk](/get-started/sdk.mdx).
+
+<Warning>
+Prem AutoPilot is only available in Pro and Business Plans. You can know more about different plans [in out credits section](/get-started/credits.mdx). 
+</Warning>
```
Comment (nsosio) [inline get-started/autopilot.mdx:13]: enhancements 2/3 available only on business

Comment (Anindyadeep) [reply get-started/autopilot.mdx:13]: I see, let me change the things written under warning

**Review by nsosio — APPROVED** [review]

**Comment by PremAIDevRel** [comment]
@Anindyadeep  Looks good so far here is some feedbak:
 Further explain the different inputs used for Auto-Pilot like Enhanced system prompt and the others. We need to make sure those are clear.

**Comment by Anindyadeep** [comment]
> @Anindyadeep Looks good so far here is some feedbak: Further explain the different inputs used for Auto-Pilot like Enhanced system prompt and the others. We need to make sure those are clear.

Hey sure, let's discuss on those enhancement and may be I can create another PR to address that :)

**Pull request merged**